### DISCUSSION TITLE
scrollbar doesn't scroll all the way for body

### DISCUSSION THREAD

**Discussion by jzhang172** [idiotWu/smooth-scrollbar#518] [Q&A]
When I try to use the plugin, I'm using the latest angular ~15, the scrollbar only scrolls a certain amount on the page.

      let body = document.body;
      Scrollbar.init(body);

**Comment by Minozzzi** [comment]
You need to set a height for your body and set options for better experience.
Like:

```css
body {
  height: 100vh;
}
```

And

```tsx
let body = document.body;
const options = {
  // your options go here
}

Scrollbar.init(body, options)
